packages/plugin-cutout-library-web/src/index.ts

```diff
@@ -6,19 +7,31 @@ export const DEFAULT_ASSET_BASE_URI = `https://staticimgly.com/${PLUGIN_NAME.rep
 )}/${PLUGIN_VERSION}/dist/assets`;
 
 export const DEFAULT_PLUGIN_CONFIGURATION = {
-  assetBaseUri: DEFAULT_ASSET_BASE_URI
+  assetBaseUri: DEFAULT_ASSET_BASE_URI,
+  createCutoutFromBlocks: (blockIds: number[], engine: CreativeEngine) => {
+    return engine.block.createCutoutFromBlocks(blockIds, 0, 2, false);
+  }
 };
+export type CreateCutoutFromBlocks = (
+  blockIds: number[],
+  engine: CreativeEngine
+) => DesignBlockId;
 
 export interface PluginConfiguration {
   assetBaseUri: string;
   ui?: {
     locations: UILocations[];
   };
+  // Can be used to customize the cutout block creation call parameters.
```

Comment:
Is this something we want to document or is this just for testing and internal for now?

Reply:
Since the parameters are really hard to grasp even for us internally, I don't think it makes sense to expose/document this publicly at this point.